modules/swagger-codegen/src/main/resources/csharp/ApiClient.mustache

```diff
@@ -145,7 +145,7 @@ namespace {{packageName}}.Client
         /// <param name="contentType">Content Type of the request</param>
         /// <returns>Object</returns>
         public Object CallApi(
-            String path, RestSharp.Method method, Dictionary<String, String> queryParams, Object postBody,
+            String path, RestSharp.Method method, List<KeyValuePair<String, String>> queryParams, Object postBody,
```

Comment:
@wing328 What's your take on the method overload comment? My concern is that `CallApi` and `CallApiAsync` are public methods. I don't doubt people are using it outside of the generated code. It could be marked internal, but that would also break those callers.